bindata/manifests/metrics-exporter/metrics-daemonset.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app.kubernetes.io/name: sriov-network-metrics-exporter
+  name: sriov-network-metrics-exporter
+  namespace: {{.Namespace}}
+  annotations:
+    release.openshift.io/version: "{{.ReleaseVersion}}"
+spec:
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: sriov-network-metrics-exporter
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: sriov-network-metrics-exporter
+    spec:
+      hostNetwork: true
+      {{- if .ImagePullSecrets }}
+      imagePullSecrets:
+      {{- range .ImagePullSecrets }}
+      - name: {{ . }}
+      {{- end }}
+      {{- end }}
+      containers:
+      - args:
+        - --path.kubecgroup=/host/kubecgroup
+        - --path.sysbuspci=/host/sys/bus/pci/devices/
+        - --path.sysclassnet=/host/sys/class/net/
+        - --path.cpucheckpoint=/host/cpu_manager_state
+        - --path.kubeletsocket=/host/kubelet.sock
+        - --collector.kubepoddevice=true
+        - --collector.vfstatspriority=sysfs,netlink
+        image: {{.Image}}
+        imagePullPolicy: Always 
+        name: metrics-exporter
+        resources:
+          requests:
+            memory: 100Mi
+            cpu: 100m
+          limits:
+            memory: 100Mi
+            cpu: 100m
+        securityContext:
+          capabilities:
+            drop:
+              - ALL      
+          readOnlyRootFilesystem: true
+          allowPrivilegeEscalation: false
+        volumeMounts:
+        - mountPath: /host/kubelet.sock
+          name: kubeletsocket
+        - mountPath: /host/sys/bus/pci/devices
+          name: sysbuspcidevices
+          readOnly: true
+        - mountPath: /host/sys/devices
+          name: sysdevices
+          readOnly: true
+        - mountPath: /host/sys/class/net
+          name: sysclassnet
+          readOnly: true
+        - mountPath: /host/kubecgroup
+          name: kubecgroup
+          readOnly: true
+        - mountPath: /host/cpu_manager_state
+          name: cpucheckpoint
+          readOnly: true
+        - name: sriov-network-metrics-exporter-config
+          mountPath: /etc/sriov-network-metrics-exporter
+      nodeSelector:
+        kubernetes.io/os: linux
+        feature.node.kubernetes.io/network-sriov.capable: "true"
+      restartPolicy: Always
+      tolerations:
+      - operator: Exists
```

Comment:
do we need this toleration here? maybe we can use the same as we have in the other daemons?